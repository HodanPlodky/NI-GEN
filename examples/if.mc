int main() {
    if (1) {
        return 1;
    }
    return 0;
}

int f() {
    int x = 1;
    if (1) {
        x = 2;
    }
    return x;
}

int g() {
    int x;
    if (1) {
        x = 5;
    }
    else {
        x = 10;
    }
    return x;
}
