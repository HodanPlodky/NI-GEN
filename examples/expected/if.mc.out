input file : examples/tests/if.mc
return 1
